src/rules/sortImportsRule.ts

```diff
@@ -0,0 +1,244 @@
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+const RULE_NAME = 'sort-imports';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static metadata: Lint.IRuleMetadata = {
+    ruleName: RULE_NAME,
+    description: 'enforce sorting import declarations within module',
+    rationale: Lint.Utils.dedent`
+            When declaring multiple imports, a sorted list of import declarations make it easier for developers to 
```

Comment:
This one has too much indentation. make it so that its only right below the `t` in rationale.